restapi/src/main/java/io/stargate/web/docsapi/resources/CollectionsResource.java

```diff
@@ -96,8 +110,9 @@ public Response getCollections(
   @ApiResponses(
       value = {
         @ApiResponse(code = 201, message = "Created"),
-        @ApiResponse(code = 409, message = "Conflict", response = Error.class),
         @ApiResponse(code = 401, message = "Unauthorized", response = Error.class),
+        @ApiResponse(code = 409, message = "Conflict", response = Error.class),
+        @ApiResponse(code = 422, message = "Unprocessable entity", response = Error.class),
```

Comment:
Where do we actually report `422`?

Reply:
Yea this is per RFC-4918, we were able to process the body, but it was not valid -> https://datatracker.ietf.org/doc/html/rfc4918#section-11.2

Reply:
I mean, where do we use this error in our code?

Reply:
ahhh -> https://github.com/stargate/stargate/pull/962/files#diff-cbe6e8c417a014d3779c711a4c0ebfa09d2ab8688fdb66020cdf4ab31500f283R46

Reply:
Thx! TIL